.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
What is the ".idea" folder used for?

Reply:
I'm using JetBrain's WebStorm IDE alongside VSCode. I would have normally not included those files in the commit but in my rush to get ready for work I selected all the files and force pushed because there was some dumb issues with this branch.

Reply:
Please don't upload the files that may not be useful for others. Please fix all issues and then invite others to peer review it. Thanks.

Reply:
I agree. Please only leave in files required for the project.